bats_test/create-drive/drive-name-validation.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+# Support lib
+load '/home/node/packages/node_modules/bats-support/load.bash'
+# Assertions
+load '/home/node/packages/node_modules/bats-assert/load.bash'
+
+@test "create drive with empty name" {
```

Comment:
These test names should not just be what the test inputs are. It should also include the expected behavior, e.g.
```
@test "create drive with empty name results in an error message and exit code 1" {
```

Reply:
[926db0a](https://github.com/ardriveapp/ardrive-cli/pull/247/commits/926db0a540bd62fbd7070aa66f12ff6f385d002e)